spec/system/admin/order_spec.rb

```diff
@@ -951,10 +951,12 @@ def new_order_with_distribution(distributor, order_cycle)
           order.finalize! # ensure order has a payment to capture
           order.payments << create(:check_payment, order:, amount: order.total)
           order.payments.first.capture!
+          login_as_admin
           visit spree.edit_admin_order_path(order)
         end
 
         it "ships the order and shipment email is sent" do
+          login_as_admin
```

Comment:
Why is this needed ? it should already be logged as an admin because it's done in the `before` block.

Reply:
These specs are in the context of an enterprise manager and we have an enterprise manager signed in this block 
```
    before(:each) do
      @enterprise_user = create(:user)
      @enterprise_user.enterprise_roles.build(enterprise: supplier1).save
      @enterprise_user.enterprise_roles.build(enterprise: coordinator1).save
      @enterprise_user.enterprise_roles.build(enterprise: distributor1).save

      login_as @enterprise_user
    end
    ```
    I had some trouble figuring out why the permission failed for the signed in enterprise user when trying to ship an order.
    So I was hoping to get help during review.